conda-recipe/run_test.bat

```diff
@@ -15,34 +15,18 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 rem ============================================================================
 
-rem %1 - scikit-learn-intelex repo root
+rem %1% - scikit-learn-intelex repo root (leave empty if it's PWD)
```

Comment:
Shouldn't this say "%cd%" since it's windows?

Reply:
I added mention of both.